idris-commands.el

```diff
@@ -238,7 +238,9 @@ line."
            (lambda (_condition)
              (when (member 'warnings-tree idris-warnings-printing)
                (idris-list-compiler-notes)
-               (pop-to-buffer (idris-buffer-name :notes)))))))
+	       (if idris-stay-in-current-window-on-compiler-error
```

Comment:
yes. the indentation here on github doesn't look right, and when I try to mark it with a mouse, it seems to be a tab character